pkg/object/objects_test.go

```diff
@@ -816,13 +813,21 @@ spec:
 
 	for _, tt := range manifestDiffWithSelectAndIgnoreTests {
 		t.Run(tt.desc, func(t *testing.T) {
-			got, err := ManifestDiffWithSelectAndIgnore(tt.yamlStringA, tt.yamlStringB, tt.selectResources, tt.ignoreResources)
-			if err != nil {
-				t.Fatalf("unexpected error: %v", err)
-			}
-			if !strings.Contains(got, tt.want) {
-				t.Errorf("%s:\ngot:\n%v\ndoes't contains\nwant:\n%v", tt.desc, got, tt.want)
-			}
+			got, err := ManifestDiffWithSelectAndIgnore(tt.yamlStringA, tt.yamlStringB,
+				tt.selectResources, tt.ignoreResources, false)
+			reportDiffErr(err, t, tt.desc, got, tt.want)
+			gotVerbose, err := ManifestDiffWithSelectAndIgnore(tt.yamlStringA, tt.yamlStringB,
+				tt.selectResources, tt.ignoreResources, true)
+			reportDiffErr(err, t, tt.desc, gotVerbose, tt.want)
 		})
 	}
 }
+
+func reportDiffErr(err error, t *testing.T, desc, got, want string) {
```

Comment:
See above. It's probably better to leave that in the original location and remove duplication as suggested.